installmacos.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/zsh
+echo "This script will install homebrew, python, git and pip (If not arleady installed!). It will also install all modules."
+
+#If no homebrew is found
+if test ! $(which brew); then
+    #Install homebrew
+    echo "Installing homebrew..."
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+fi
+#If no python is found
+if test ! $(which python3); then
+    #Install python
+    echo "Installing python..."
+    brew install python@3.10
+fi
+#If no git is found
+if test ! $(which git); then
+    #Install git
+    echo "Installing git..."
+    brew install git
+fi
+
+#Clone the repo
+git clone https://github.com/ThioJoe/YT-Spammer-Purge.git
```

Comment:
I think you should only go until the latest tag (normal or prerelease) as there could be dangling commits that might not be ready for use that could break something potentially.

Reply:
Ok, working on that.

Reply:
i'm stumped, tring to find a solution, but cant. Any help?